src/layout/VerifyEmailWarning.js

```diff
@@ -0,0 +1,37 @@
+import i18n from '@dhis2/d2-i18n'
+import { NoticeBox } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export function VerifyEmailWarning({ config }) {
+    const enforceVerifiedEmail =
+        config?.system?.settings?.enforceVerifiedEmail || false
```

Comment:
this should be `config?.system?.settings?.settings?.enforceVerifiedEmail`, right?

Also, I don't think `config` can be undefined as this component will not be rendered if config (this.context?.d2) is undefined, so you could get rid of the first level of optional chaining, i.e. `config.system?.settings?.settings?.enforceVerifiedEmail`